.clang-format

```
@@ -59,8 +59,10 @@ ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
-IncludeBlocks:   Preserve
+IncludeBlocks:   Regroup
 IncludeCategories:
+  - Regex:           '<[[:alnum:]._]+>'
+    Priority:        5
```

```suggestion
  - Regex:           '<[[:alnum:]._]+>'
    Priority:        5
  - Regex:           '^(<"podio/)'
    Priority:        1
```

To keep the podio ones at the top?